client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1986,6 +1987,13 @@
                         }
 
                         _.extend(item, updates);
+
+                        if (item.profile) {
+                            content.getType(item.profile)
+                            .then(function(type) {
+                                scope.contentType = type;
```

Comment:
imo it would be good to assign also `scope.schema` and use it in template instead of `contentType.schema...` like that, we can have some default schema in case there is no content type for profile